src/customize-pop/dock-settings.md

```diff
@@ -0,0 +1,103 @@
+# Dock Settings
+
+This section describes settings located in the `Dock` tab for modifying Dock visibility behavior and Dock icons.
+
+![Customize Dock](/images/customize-desktop/customize-dock.png)
+
+## Enable Dock
+
+Disable or enable the Dock's visibility.
+
+**Dock Enabled**
+
+![Dock Enabled](/images/customize-desktop/dock-enabled.png)
+
+**Dock Disabled**
+
+![Dock Disabled](/images/customize-desktop/dock-disabled.png)
+
+## Dock Options
+
+Dock options include settings to enable icons for utilities (such as the Launcher, Applications menu, and viewing workspaces), configure focusing behavior when an application's Dock icon is clicked, and enabling visual characteristics of the Dock.
+
+![Dock Options](/images/customize-desktop/dock-options.png)
+
+### Extend Dock to Edges of the Screen
+
+Lengthen the Dock's edges so that it appears as a panel that extends to the screen edges.
+
+![Extend Dock Edges](/images/customize-desktop/extend-dock-edges.png)
+
+### Show Launcher Icon in Dock
+
+Display an icon that starts the Launcher when clicked.
+
+![Launcher Icon](/images/customize-desktop/launcher-icon.png)
+
+### Show Workspaces Icon in Dock
+
+Display an icon that activates the workspaces and windows view when clicked.
+
+![Workspaces Icon](/images/customize-desktop/workspaces-icon.png)
+
+### Show Applications Icon in Dock
+
+Display an icon that launches the Application menu when clicked.
+
+![Applications Icon](/images/customize-desktop/applications-icon.png)
+
+### Show Mounted Drives
+
+Display an icon for each mounted drive.
+
+![Connected Drive Icon](/images/customize-desktop/connected-drive-icon.png)
+
+### Icon Click Action
+
+Specify what action should occur when an application's Dock icon is clicked. See examples of Icon Click Action behavior in [Navigate Between Running Applications](/navigate-pop/switching-apps.md#using-the-dock).
+
+![Icon Click Action](/images/customize-desktop/icon-click-action.png)
+
+| Option | Function |
+|:----------|:----------|
+| Launch or Cycle Windows (Default) | Focus a running application when its icon is clicked. Additional clicks cycle through instances of that application. |
+| Launch or Minimize Windows | Focus or minimize a running application when its icon is clicked. |
```

Comment:
I really like how this section is currently reading for the most part. You might consider specifying `Focus or minimize all windows of a running application...` to clarify that the reason one might not want this is the lack of ability to focus a specific window.

Reply:
Clarified